HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,527 @@
+
+This HIP extends the active HIP 412 by using its standard scheme for non-fungible token metadata on HTS. NFTs minted using a more granular standard allow for better metadata parsing and reduce implementation nuances due to different interpretations of HIP 412.
+
+## Motivation
+
+NFT creators desire a more granular standard to define their non-fungible token metadata on HTS. The current implementation defined by HIP 412 allows for different interpretations. It's a risk for the Hedera NFT community because users will create different metadata implementations. Therefore, making it harder for NFT tooling to interpret this data correctly. 
+
+The goal is that this HIP defines a granular standard that the entire Hedera NFT community can adopt to set us up for the future.
+
+## Rationale
+
+This token metadata standard has been developed with input from multiple developers and projects in the NFT space. The goal is to provide a more granular NFT standard for "Web3 NFTs", providing a more concrete implementation for HIP-412. 
+
+This HIP focuses on:
+
+1. Providing strict guidelines for artists to define their NFT token metadata
+2. Maintaining flexibility via the `properties` object allowing for non-specified fields
+3. Maintaining a good level of interoperability and coherence with other standards such as OpenSea
+4. Providing sufficient examples for all types of NFTs to find the correct metadata implementation as an artist
+  
+## Specification
+
+Below is the human-readable schema, presented to maximize clarity. 
+
+```json
+{
+    "name": "NFT name (required)",
+    "creator": "artist (required)",
+    "creatorDID": "DID URI (optional)",
+    "description": "human readable description of the asset (recommended)",
+    "image": "cid or path to the NFT's preview image file (required)",
+    "image_integrity": "SHA-256 digest of the file pointed by the image field (recommended)",
+    "type": "MIME type (required)",
+    "format": "standard specification (required - i.e. none, opensea, HIPXXX, hashaxis)",
+    "properties": {
+        // json object that cover the overarching properties of the token
+        "files": [
+            {
+                "uri": "uri to file (required)",
+                "sha256_checksum": "cryptographic hash of the representation of the resource the author expects to load (recommended)",
+                "type": "MIME type (required)",
+                "is_default_file": "(Type: boolean) indicates if the file is the main file for this NFT (conditionally optional - required when multiple files are listed)",
+                "metadata": "metadata object (optional)",
+                "metadata_uri": "uri to metadata (optional)",
+                "localization": [
+                    // optional localization array
+                    {
+                        "uri": "uri to file (required)",
+                        "locale": "language identifier (required)"
+                    }
+                ]
```

Comment:
localization should appear at the root of the metadata as well so that the main name, description etc can be localized.

Reply:
Right, didn't think of that!